### PR TITLE
Subtitle applied only to edition 

### LINKED ISSUE #967: Subtitle in work's title should not be applied to editions if no edition subtitle present
The subtitle is not usually applicable to all editions. Translations in particular. Example: https://openlibrary.org/books/OL26414568M/Farbenblind

### PR DESCRIPTION
**Pull request by tabshaikh**
Closes #967 

### Evidence

Before:
Screenshot from 2020-01-15 23-29-12
After:
Screenshot from 2020-01-15 23-29-41

### Stakeholders

@mekarpeles

### CODE REVIEW

File: openlibrary/templates/type/edition/view.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 $ ocaid = page.get('ocaid')
 $ book_title = page.get('title', '') if page.title else (work.title if work else '')
 $ work_title = work.title if work else page.get('title', '')
-$ book_subtitle = page.subtitle if page.get('subtitle') else (work.subtitle if work and work.get('subtitle') else '')
```
Comment (cdrini) [inline openlibrary/templates/type/edition/view.html]: Can simplify this to:

```py
$ book_subtitle = page.get('subtitle', '')
```

**Comment by mekarpeles** [comment]
I'm not positive this issue is t moving is in the right direction. If like is to hold off on meeting until Thursday's discussion about book pages.

**Comment by tabshaikh** [comment]
@mekarpeles @cdrini are we moving forward with this pr or should i close it :)

**Comment by tfmorris** [comment]
This PR only addresses the case where there's no edition subtitle and the work subtitle is used in its place. There's a good discussion on #967 concerning why attempting to merge fields across records is a) hard and b) generally a bad idea. 

I think there was consensus there that this PR would be an improvement.

**Pull request merged**